var/spack/repos/builtin/packages/fms/package.py

```diff
@@ -119,13 +110,8 @@ class Fms(CMakePackage):
     depends_on("netcdf-c")
     depends_on("netcdf-fortran")
     depends_on("mpi")
-    depends_on("llvm-openmp", when="+openmp %apple-clang", type=("build", "run"))
     depends_on("libyaml", when="+yaml")
-
-    # DH* 20220602
-    depends_on("ecbuild", type=("build"), when="@release-jcsda")
-    depends_on("jedi-cmake", type=("build"), when="@release-jcsda")
-    # *DH 20220602
+    depends_on("llvm-openmp", when="+openmp %apple-clang", type=("build", "run"))
```

Comment:
Note this line remains intact, it's just moved back to where it was in the authoritative repo version.